src/common/util/metrics.h

```diff
@@ -13,16 +13,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#ifndef SRC_SERVER_UTIL_METRICS_H_
```

Comment:
any reason to move the header to common folder? seems there're a few files affected. new added file would be used in other components?

Reply:
Vanilla v6d only enables metrics on the server side, so metrics.h/.cc are put under server's code folder. Now, this PR enables metrics for both client and server, so metrics.h/.cc have been moved into the common folder.